cli/build.go

```diff
@@ -46,7 +46,7 @@ func (b *BuildCmd) SetFlags(f *flag.FlagSet) {
 	f.BoolVar(&b.CleanBuild, "clean", false, "Perform a completely clean build -- don't reuse anything when building")
 }
 
-func (b *BuildCmd) Execute(_ context.Context, f *flag.FlagSet, _ ...interface{}) subcommands.ExitStatus {
+func (b *BuildCmd) Execute(buildCtx context.Context, f *flag.FlagSet, _ ...interface{}) subcommands.ExitStatus {
```

Comment:
nit: `Context` parameters are by convention called `ctx`. (here and elsewhere) https://github.com/golang/go/wiki/CodeReviewComments#contexts